integration-tests/smallrye-config/src/main/resources/application.properties

```diff
@@ -3,6 +3,7 @@ configurable.source.init=database
 
 # additional locations
 smallrye.config.locations=config.properties
+smallrye.config.source.file.locations=target/classes/smallrye-config-source-file-system
```

Comment:
On second thought, maybe it's better to place it in `src/main/smallrye-config-source-file-system` so that it's not packaged?
I mean it doesn't really make a difference but it might be clearer.

Reply:
Update pushed, I prefer it this way now.